src/main/config/cromwell.conf.ctmpl

```diff
@@ -0,0 +1,57 @@
+{{with $environment := env "ENVIRONMENT"}}{{$keyname := printf "secret/dsde/%s/cromwell/cromwell.conf" $environment}}{{with vault $keyname}}
+
+include "application.conf"
+
+backend {
+  backend = "JES"
+
+  jes {
+    applicationName = "cromwell"
+    project = "{{.Data.jes_project}}"
+    baseExecutionBucket = "{{.Data.jes_base_execution_bucket}}"
+    endpointUrl = "{{.Data.jes_endpoint_url}}"
+    dockerhubCredentialsPath = "{{.Data.jes_dockerhub_credentials_path}}"
+    authenticationMode = "{{.Data.jes_authentication_mode}}"
```

Comment:
This is not part of the configuration anymore. 
This field is not in application.conf in cromwell and is not used.
We can re-instate it to make a clearer distinction between "firecloud mode" and "gitc mode" but as of now it is not read by cromwell.